src/Microsoft.AspNetCore.Localization/RequestCultureProvider.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.AspNetCore.Localization
     public abstract class RequestCultureProvider : IRequestCultureProvider
     {
         /// <summary>
+        /// Result that indicates that this instance of <see cref="RequestCultureProvider" /> could not determine the
+        /// request culture.
+        /// </summary>
+        protected static readonly Task<ProviderCultureResult> NullResult = Task.FromResult(default(ProviderCultureResult));
```

Comment:
Hmm this is probably fine. Can't do any harm really. Or maybe it should have a more specific name, though? Is `NullProviderCultureResult` too long? I think it reads better in the code, though.